frontend/src/api.ts

```diff
@@ -162,335 +163,350 @@ axios.interceptors.request.use(jwtRequestInterceptor);
+    async explainText(modelId: string, datasetId: string, inputData: object, featureName: string) {
+        return apiV2.post<unknown, ExplainTextResponseDTO>(
+            `/models/explain-text/${featureName}`,
+            {
+                features: inputData,
+            },
+            {params: {modelId, datasetId}}
+        );
+    },
+    // feedbacks
+    async submitFeedback(payload: CreateFeedbackDTO, projectId: number) {
+        return apiV2.post<unknown, void>(`/feedbacks/${projectId}`, payload);
+    },
+    async getProjectFeedbacks(projectId: number) {
+        return apiV2.get<unknown, FeedbackMinimalDTO[]>(`/feedbacks/all/${projectId}`);
+    },
+    async getFeedback(id: number) {
+        return apiV2.get<unknown, FeedbackDTO>(`/feedbacks/${id}`);
+    },
+    async replyToFeedback(feedbackId: number, content: string, replyToId: number | null = null) {
+        return apiV2.post<unknown, void>(`/feedbacks/${feedbackId}/reply`, <CreateFeedbackReplyDTO>{
+            content,
+            replyToReply: replyToId,
+        });
+    },
+    async deleteFeedback(id: number) {
+        return apiV2.delete<unknown, void>(`/feedbacks/${id}`);
+    },
+    async deleteFeedbackReply(feedbackId: number, replyId: number) {
+        return apiV2.delete<unknown, void>(`/feedbacks/${feedbackId}/replies/${replyId}`);
+    },
+    async runAdHocTest(projectId: number, testUuid: string, inputs: Array<FunctionInputDTO>) {
+        return apiV2.post<unknown, TestTemplateExecutionResultDTO>(`/testing/tests/run-test`, {
+            projectId,
+            testUuid,
+            inputs,
+        });
+    },
+    async getCatalog(projectId: number) {
+        return apiV2.get<unknown, CatalogDTO>(`/catalog`, {
+            params: {
+                projectId,
+            },
+        });
+    },
+    async createSlicingFunction(comparisonClauses: Array<ComparisonClauseDTO>) {
+        return apiV2.post<unknown, SlicingFunctionDTO>(`/slices/no-code`, comparisonClauses);
+    },
+    async uploadLicense(form: FormData) {
+        return apiV2.post<unknown, unknown>(`/ee/license`, form, {
+            headers: {
+                'Content-Type': 'multipart/form-data',
+            },
+        });
+    },
+    async finalizeSetup(allowAnalytics: boolean, license: string) {
+        return apiV2.post<SetupDTO, unknown>(`/setup`, {
+            allowAnalytics: allowAnalytics,
+            license: license,
+        });
+    },
+    async datasetProcessing(projectId: number, datasetUuid: string, functions: Array<ParameterizedCallableDTO>, sample: boolean = true) {
+        return apiV2.post<unknown, DatasetProcessingResultDTO>(
+            `/project/${projectId}/datasets/${encodeURIComponent(datasetUuid)}/process?sample=${sample}`,
+            functions
+        );
+    },
+    async getPushes(modelId: string, datasetId: string, idx: number, features: any) {
+        return apiV2.post<unknown, unknown>(`/pushes/${modelId}/${datasetId}/${idx}`, {
```

Comment:
Why do you use Post request to get elements?

Why not specifying the type of the return object?